docs/content/how-to-develop-a-worker-program-in-node.md

```diff
@@ -15,45 +15,45 @@ Below is an example of starting a Worker that polls the Task Queue named `tutori
-| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `activities`      | Mapping of activity name to implementation. Automatically discovered from ${workDir}/activities if workDir is provided.                                            |
-| `nodeModulesPath` | Path for webpack to look up modules in for bundling the Workflow code. Automatically discovered if `workDir` is provided. Defaults to `${workDir}/../node_modules` |
-| `workflowsPath`   | Path to look up workflows in. Automatically discovered if `workDir` is provided. Defaults to `${workDir}/workflows`                                                |
-| `dataConverter`   | placeholder for future DataConverter feature (pending feature)                                                                                                     |
-| `dependencies`    | Allows injection of external dependencies (Advanced feature: see [External Dependencies](/docs/node/external-dependencies))                                        |
-| `interceptors`    | A mapping of interceptor type to a list of factories or module paths (Advanced feature: see [Interceptors](/docs/node/interceptors))                               |
+| Options            | Description                                                                                                                          |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
+| `activities`       | Mapping of Activity name to implementation.                                                                                          |
+| `nodeModulesPaths` | Paths for webpack to look up modules in for bundling the Workflow code.                                                              |
+| `workflowsPath`    | Path to look up Workflows in.                                                                                                        |
+| `dataConverter`    | placeholder for future DataConverter feature (pending feature)                                                                       |
+| `dependencies`     | Allows injection of external dependencies (Advanced feature: see [External Dependencies](/docs/node/external-dependencies))          |
+| `interceptors`     | A mapping of interceptor type to a list of factories or module paths (Advanced feature: see [Interceptors](/docs/node/interceptors)) |
 
 For example, if you are working in monorepo style and want `node_modules` at your project root, with all Temporal code inside a `/temporal/src` folder, you can force `nodeModulesPath`:
 
 ```ts
-// this file is /temporal/src/worker.ts but node modules are at /node_modules
```

Comment:
This entire sample is irrelevant now, should we just drop it? Do you have a better example in mind?